de.bforartists.Bforartists.json

```diff
@@ -0,0 +1,162 @@
+{
+    "id": "de.bforartists.Bforartists",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "bforartists",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=host",
+        "--filesystem=/run/spnav.sock:ro",
+        "--env=SPNAV_SOCKET=/run/spnav.sock",
+        "--env=TMP_DIR=/tmp",
+        "--env=TMP=/tmp"
+    ],
+    "add-extensions": {
+        "de.bforartists.bforartists.Codecs": {
+            "directory": "lib/codecs",
+            "add-ld-path": "lib",
+            "bundle": true,
+            "autodelete": true
+        }    
+    },
```

Comment:
What's the point of that? There's no reason to split codecs to separate extension, unless explicitly requested by some US-based entity interested in shipping this app by default with their distro. Not a blocker, just curious why you did that.

Reply:
It was found for the Blender flatpak that Freedesktop's FFMPEG doesn't play well with Blender (namely OpenH264). Bforartists is a Blender fork with UI changes, so what affects Blender also affects Bforartists